src/Orchard.Web/Modules/Orchard.ContentTypes/Views/DisplayTemplates/EditTypeViewModel.cshtml

```diff
@@ -9,7 +9,7 @@
     <div class="properties">
         <h3>@Model.DisplayName</h3> @if (!string.IsNullOrWhiteSpace(stereotype)) { <text><span class="stereotype" title="Stereotype">- @stereotype</span></text>}
         @if (creatable) {
-        <p class="pageStatus">@Html.ActionLink(T("Create New {0}", Model.DisplayName).Text, "Create", new {area = "Contents", id = Model.Name})</p>
+        <p class="pageStatus">@Html.ActionLink(T("Create New {0}", new HtmlString(Model.DisplayName)).Text, "Create", new {area = "Contents", id = Model.Name})</p>

```

Comment:
Right, what we do everywhere else is to create a link directly using `<a href="Url.Action ...">@T(...)</a>`. You could add the overload with `IHtmlString` but it might too much code. Up to you.